database/index.js

```diff
@@ -29,36 +44,27 @@ const getReviewsByProductId = function (productid, callback) {
   });
 };
 
-// create review for one product
-const postReviewsByProductId = function (product, callback) {
-  const queryString = 'Insert into reviews (product_id, title, text, date, author, overall_rating, value_rating, quality_rating, appearance_rating, ease_of_assembly_rating, works_as_expected_rating, recommended, helpful_count, not_helpful_count) VALUES(?, ?, ?, ?,?, ?, ?, ?,?, ?, ?, ?,?, ?)';
-
-  connection.query(queryString, (Object.values(product)), (error, results, fields) => {
+// update
+const incrementReviewCounts = function (req, callback) {
+  connection.query('UPDATE reviews SET ?? = ?? + 1 WHERE id = ?', [req.column, req.column, req.id], (error, results, fields) => {
     if (error) {
-      console.log(error);
+      return error;
     }
     callback(null, results);
   });
 };
+
+
 // delete one review instead all review for one product
 const deleteReviewsById = function (id, callback) {
   const queryString = `delete from reviews where id =${connection.escape(id)}`;
```

Comment:
what's the purpose of connection.escape() here, or was that there already?

Reply:
No, I usually add .escape() so that i have access to the value of id but I guess the ${} does the job. thanks airbnb eslint!